kratos/geometries/geometry.h

```diff
@@ -1122,41 +1122,102 @@ class Geometry : public PointerVector<TPointType>
         return false;
     }
 
+    ///@}
+    ///@name Boundaries
+    ///@{
+
+    /**
+     * @brief This method gives you all boundaries entities of this geometry.
+     * @details This method will gives you all the boundaries entities
+     * @return GeometriesArrayType containes this geometry boundaries entities.
+     * @see GeneratePoints()
+     * @see GenerateEdges()
+     * @see GenerateFaces()
+     */
+    virtual GeometriesArrayType GenerateBoundariesEntities( void )
```

Comment:
```suggestion
    virtual GeometriesArrayType GenerateBoundariesEntities
```

why `(void)` ?

Reply:
It was in Edges and Faces, it does exactly the same

Reply:
I would suggest to remove it, looks to me like legacy
Or is it used/required? I didn't even know that this was valid C++ syntax

Reply:
Is very C:

https://en.wikipedia.org/wiki/Void_type

![imagen](https://user-images.githubusercontent.com/19991680/57132473-02c26500-6da0-11e9-9c2a-1364dd7090af.png)

I will change it